src/types.ts

```diff
@@ -211,3 +211,20 @@ export type AppClient = {
    */
   requirePkce: boolean;
 };
+
+export type UserInfo = {
+  name: string;
+  middleName?: string;
+  givenName?: string;
+  phoneNumber?: string;
+  phoneNumberVerified?: boolean;
+  email?: string;
+  emailVerified?: boolean;
+  familyName?: string;
+  birthDate?: string;
+  address?: string;
```

Comment:
`address` should use the format from the spec in ticket #623

Reply:
Did you mean #624 ?

Reply:
Yes! oops